test/Tests/README.md

```diff
@@ -0,0 +1,5 @@
+This is a set of tests for the `Base` library for Enso.
+
+The run test suite for HTTP component requires adctive `httbin` server on the
+localhost. If it is present, the port it listens to should be provided by the
+`ENSO_HTTP_TEST_HTTPBIN_URL` for value like `http://localhost:8080`.
```

Comment:
```suggestion
The run test suite for the HTTP component requires an active `httbin` server on the
localhost. If it is present, the port it listens to should be provided by setting the
`ENSO_HTTP_TEST_HTTPBIN_URL` environment variable to a value like `http://localhost:8080`. The URL should not contain a trailing slash.
```

(modulo prettier), so I guess it would need to be:

```suggestion
The run test suite for the HTTP component requires an active `httbin` server on
the localhost. If it is present, the port it listens to should be provided by
setting the `ENSO_HTTP_TEST_HTTPBIN_URL` environment variable to a value like
`http://localhost:8080`. The URL should not contain a trailing slash.
```